openquake/hazardlib/geo/utils.py

```diff
@@ -199,6 +202,54 @@ def assoc(objects, sitecol, assoc_dist, mode):
             objects, assoc_dist, mode)
 
 
+ERROR_OUTSIDE = 'The site (%.1f %.1f) is outside of any vs30 area.'
+
+
+def assoc_to_polygons(polygons, data, sitecol, _assoc_dist, mode):
+    """
```

Comment:
Why the underscore in front of `_assoc_dist`?

Reply:
to denote an unused argument, but actually I realize that I don't have to keep that argument in that function, going to remove it.